src/westminster/scripts/westminster_train_folds.py

```diff
@@ -255,14 +272,14 @@ def main():
                 cmd += "conda activate %s;" % options.conda_env
                 cmd += " echo $HOSTNAME;"
 
-                cmd += " hound_train.py"
+                cmd += " %s" % train_cmd
                 cmd += " %s" % options_string(options, train_options, rep_dir)
                 cmd += " %s %s" % (params_file, " ".join(rep_data_dirs))
 
                 name = "%s-train-f%dc%d" % (options.name, fi, ci)
                 sbf = os.path.abspath("%s/train.sb" % rep_dir)
-                outf = os.path.abspath("%s/train.out" % rep_dir)
-                errf = os.path.abspath("%s/train.err" % rep_dir)
+                outf = os.path.abspath("%s/train.%%j.out" % rep_dir)
```

Comment:
This would break a lot of my model analysis code. Do we need it? I've never had multiple training logs in the same directory. If I want a new training run, I make a new directory.

Reply:
i changed this back!